docs/getting-started/integrations-cdn/react-default-cdn.md

```diff
@@ -38,7 +38,238 @@ You get ready-to-use code tailored to your needs!
 
 This guide assumes you have a React project. You can create a basic React project using [Vite](https://vitejs.dev/). Refer to the [React documentation](https://react.dev/learn/start-a-new-react-project) to learn how to set up a project in the framework.
 
-## Component properties
+### Installing from npm
+
+Install the `@ckeditor/ckeditor5-react` package:
+
+```bash
+npm install @ckeditor/ckeditor5-react
+```
+
+Use the `<CKEditor>` component inside your project. The below example shows how to use the component with the open-source plugins.
+
```

Comment:
I miss the info what `useCKEditorCloud` does (adding all needed scripts to a page).